include/lsst/afw/math/GaussianProcess.h

```diff
@@ -603,6 +603,42 @@ class GaussianProcess {
                     std::shared_ptr< Covariogram<T> > const &covarIn);
 
     /**
+     * @brief return the number of data points stored in the GaussianProcess
+    */
+    int getPoints() const;
```

Comment:
Can we use getNPoints() or something like that perhaps?  getPoints sounds too much like getData to me.

I suppose this applies to the private variable `_pts` as well: -> `_npts` or something.